brave-lists/debounce.json

```diff
@@ -364,5 +364,16 @@
     "prepend_scheme": "https",
     "action": "regex-path",
     "param": "^/navigation-tracking/(.*);.html$"
+  },
```

Comment:
here is a very very minor request, but could you move the new rules _before_ the internal / brave testing ones?  Its a super minor nit kind of thing, but would just be nice to not have the brave-QA-testing rules in the middle of "real world" rules